docs/content/api/errors.en.md

```diff
@@ -12,49 +12,52 @@ See the [error protocol buffers](https://github.com/trisacrypto/trisa/blob/main/
 
 ## TRISA Errors Codes, Messages, and Descriptions
```

Comment:
Updated the error codes chart to ensure it was up to date with the latest protocol buffers.